### ISSUE TITLE
[docs] Find out and note minimal supported browser versions

### ISSUE THREAD

**Issue by AndreyBelym** [DevExpress/testcafe#5228]
Users try to use browsers that hammerhead cannot handle (especially with BrowserStack). We need to find minimal browser versions that we support and note them somehow.

**Comment by Archish27** [comment]
@AndreyBelym I have created PR that will filter chrome & firefox browsers > 53.0 versions as the older versions seem to be breaking at BrowserStack.
https://github.com/DevExpress/testcafe-browser-provider-browserstack/pull/119

**Comment by Ogurecher** [comment]
Hello @Archish27,

Thank you for sharing your pull request. We've examined the changes you offered and the reasoning behind them. This thread was created to address the same issue in a different way. We believe that documenting the oldest supported browser versions is a more practical and flexible solution to this problem.

**Comment by Archish27** [comment]
@Ogurecher / @AndreyBelym  I tried running sessions for chrome < 53 versions it doesn't seem to work seamlessly.  Any next steps? Or changes need to be done?

**Comment by miherlosev** [comment]
The 'list-browsers' command outputs a list of all available browsers. To prevent a user from running TestCafe on an unsupported browser version, we have plans to document the minimal supported browser version. So, our next steps will be to find out the minimum versions and document them.

**Comment by Archish27** [comment]
@miherlosev I tried running few sessions on [firefox](https://automate.browserstack.com/dashboard/v2/public-build/TGx2andBbWxZUExwNExhR0lteTRsZHcwUmZYQllEUThBdHY2RlVuZGtHcz0tLVkycW1Wak5NOXJGNjUya1J4SDdlcWc9PQ==--b67b4f3f8da3c3021003f025f26206b856b56992) & [chrome](https://automate.browserstack.com/dashboard/v2/public-build/eDVUSDdjUWp5WXBFODZJRVJJdG0zd0p1djlUYTBNd3ExY0Y3cTFqL0haST0tLSt2Y1VkWURaWnVob0J1TnI2cE1XZVE9PQ==--b57112560632e58915708db9456372327b6b45aa) < 53.0 versions, the test failing were causing browser disconnected. I hope these sessions helps. 

Any help required from my side let me know. 
Thanks!

**Comment by arubtsov** [comment]
@Archish27 Thank you for the assistance. We will need to determine the minimal supported version for Internet Explorer, Microsoft Edge, Safari, and mobile browsers as well.

**Comment by Archish27** [comment]
Any ETA on this? Need to roll out soon...

**Comment by AndreyBelym** [comment]
We had plans to document supported browser versions, but having discussed this for numerous times we decided to refuse from this idea and keep up with modern web policies and evergreen browsers. We want to be fair here - testing for unsupported legacy browser versions to determine the minimal supported version seems to be too costly. Thus, TestCafe supports only those evergreen browser versions that are supported by browser vendors. For Chrome it means that the latest browser versions of supported Chrome release channels (stable, beta, dev, canary) are supported.

Stable use of older browser versions is not guaranteed. We decided to focus on using browser feature detection to detect unsupported browsers and display a descriptive error: [https://github.com/DevExpress/testcafe/issues/2908](https://github.com/DevExpress/testcafe/issues/2908). Also, since we cannot determine the exact minimal supported version, we cannot filter or restrict supported browser versions on cloud platforms like Browserstack until the platform provides some way to perform filtering on its side.

We apologize for misleading you and hope you will understand our position.
